applications/luci-app-tailscale/root/usr/share/rpcd/acl.d/luci-app-tailscale.json

```diff
@@ -0,0 +1,19 @@
+{
+	"luci-app-tailscale": {
+		"description": "Grant access to Tailscale configuration",
+		"read": {
+			"file": {
+				"/sbin/ip -s -j ad": [ "exec" ],
+				"/sbin/logread -e tailscale": [ "exec" ],
+				"/usr/sbin/tailscale": [ "exec" ]
```

Comment:
Perhaps this also should be limited to only allow specific commands instead of everything.